addons/stock/tests/test_report.py

```diff
@@ -57,7 +57,7 @@ def test_reports(self):
         target = '\n\n\n^XA\n^FO100,50\n^A0N,44,33^FD[C418]Mellohi^FS\n^FO100,100\n^A0N,44,33^FDLN/SN:Volume-Beta^FS\n^FO100,150^BY3\n^BCN,100,Y,N,N\n^FDVolume-Beta^FS\n^XZ\n\n\n'
 
         rendering, qweb_type = report._render_qweb_text(lot1.id)
-        self.assertEqual(target, rendering.replace(' ', ''), 'The rendering is not good')
+        self.assertEqual(target, rendering.decode().replace(' ', ''), 'The rendering is not good')
```

Comment:
Alternative: revert the update from 7df343dd1be1f286493660ed8fd28669ab4a8311, make `target` and the two parameters of `replace` back into bytes literals.

Reply:
Done